cloudformation/custom_resource/cluster.yaml

```diff
@@ -304,6 +332,76 @@ Resources:
       Layers:
         - !Ref PclusterLayer
 
+  CleanupS3bucketFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Timeout: 60
+      Code:
+        ZipFile: !Sub |
+          import boto3
+          import logging
+          import sys
+          from botocore.config import Config
+          logger = logging.getLogger()
+          logger.setLevel(logging.INFO)
+          
+          crhelper_path = "/opt/python/pcluster/resources/custom_resources/custom_resources_code"
```

Comment:
I don't think we need the cr helper for this -- we can just check if the event is a delete operation and then do the delete.

Reply:
Done